### ISSUE TITLE
reviewing interest field missing

### ISSUE THREAD

**Issue by advsci** [Vitaliy-1/oldGregg#36]
while registering new user, after checking reviewer role should load reviewing interest input field.

**Comment by Vitaliy-1** [comment]
Yep. And I don't see an easy way of adding such option, for now.

**Comment by Vitaliy-1** [comment]
I have made a topic on OJS forum. With help of PKP team solving this issue would be much quicker:
https://forum.pkp.sfu.ca/t/support-for-reviewing-interest-field-on-registration-page-for-the-ojs-theme/38570
